### ISSUE TITLE
Improvement in Inquiry Module | Dillon Denha | Phone Clinic - Novi

### ISSUE THREAD

**Issue by fahadimran303** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1394]
**Hi Sir**,

**Business Name**:
Phone Clinic - Novi

**Customer Stature**:
4 Store Location with Monthly Plan on Professional Package.
High Presence on Social Media.Can help us a lot in other verticals as well.

**Module**:
Inquiries

**Improvement**:

The customer came up with a request that once an inquiry is approved the system should mark an event on the admin's calendar associated with the admin's registered email address automatically.

For example: If the admin creates an inquiry for any of his customers and once the customer approves that then the admin has two options to go with moving forward which are Create a Ticket and Create an Estimate. So the customer made a request that there should be a third option beside the two available options that is Create an Appointment which will send a calendar invite on the admin's email address and will create an event for that specific day and time for which the Appointment is Booked.

**Source**:
Facebook

_Thanks and Regards_,
**Faisal Ali**

**Comment by fahadimran303** [comment]
Jira link:
https://repairdesk.atlassian.net/browse/REP-3010

**Comment by Uzair-RepairDesk** [comment]
Hi @fahadimran303 

At present, We have decided not to release new features on the iPad and web app for the next few months as one of the feedback we constantly receive is to simplify RepairDesk, make it more stable, and remove unnecessary options.
With that in mind, we've decided to pause "new feature development" and over the course of the next few months our entire team will be working on "RepairDesk 2.0"
It's going to be clean, fast, and reliable! I hope you'll like it!

Regards,
